distribution/Dockerfile-ubi-packer

```diff
@@ -0,0 +1,7 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+
+RUN curl --location --output /etc/yum.repos.d/hashicorp.repo https://rpm.releases.hashicorp.com/RHEL/hashicorp.repo
+RUN microdnf install python3 rust cargo python3-devel openssl-devel
+RUN pip3 install setuptools-rust
+RUN pip3 install ansible
```

Comment:
Subscribing the container and using Red Hat Ansible might be better. This is using `docker` so subscribing the container should be totally possible (with `podman`, more hacks are needed because it tries to be smart about subscriptions).

Reply:
This depends on the host afaik . I guess you can get around the automatic secrets mapping with mounting over the right things (/run/secrets?). I think `docker` does this automatic subscription mapping as well though, at least if the host is RHEL.